salt/kubelet/node_labels.jinja

```diff
@@ -0,0 +1 @@
+KUBELET_NODE_LABELS="--node-labels={{ node_labels|join(",") }}"
```

Comment:
> It may be possible to use a systemd drop in unit rather than replace the whole file / edit the package? Combined with something like this? https://gist.github.com/nickjacob/9909574

I've seen that solution in the past and I think it is too dirty for my taste... I think it would be better to add a envvar to the [kubelet service](https://build.opensuse.org/package/view_file/Virtualization:containers/kubernetes/kubelet.service?expand=1) file...